tic-tac-toe/src/urbit/mar/toe/player.hoon

```diff
@@ -0,0 +1,13 @@
+::
+::::  /hoon/player/toe/mar
+  ::
+/?    310
+/-    toe
+=,    toe
```

Comment:
Note that you can do `/-  *toe` to have the `=,` happen automatically.

Reply:
Worth noting, `/-  *toe` is strictly superior here. Using `=,` in this way will leave you with two paths to the structure in your subject (with and without the `toe` face), while `*toe` will simply import the structure without the face.